content/docs/command-reference/plots/diff.md

```diff
@@ -46,7 +46,7 @@ please see `dvc plots`.
 
 ## Options
 
-- `--targets <path>` - metrics file to visualize. Shows all plots by default.
+- `--targets <path>` - metric files to visualize. Shows all plots by default.
```

Comment:
`Shows all plots by default.` - Reads data from all plot files defined in `dvc.yaml` by default.

Reply:
I did add this info recently but to the command descriptions. See 91699677

I think that's better, no?